pkg/cosign/keys_test.go

```diff
@@ -48,7 +130,66 @@ func TestLoadECDSAPrivateKey(t *testing.T) {
 	if _, err := rand.Read(buf[:]); err != nil {
 		t.Fatal(err)
 	}
-	if _, err := LoadECDSAPrivateKey(buf[:], []byte("wrong")); err == nil {
+	if _, err := LoadPrivateKey(buf[:], []byte("wrong")); err == nil {
 		t.Error("expected error decrypting key!")
 	}
 }
+
+func TestImportPrivateKey(t *testing.T) {
```

Comment:
Hey, while you are fixing this test, please use a struct to hold the different test cases instead of copying and pasting the test block where you write the file, import key pair, and test the error. 

See this for example
https://github.com/sigstore/cosign/blob/1b1cafccd5dac3a44ff3aabee683a6488bcc2584/cmd/cosign/cli/dockerfile/verify_test.go#L24

(or https://gobyexample.com/testing, table-driven style)